packages/driver/cypress/e2e/dom/visibility.cy.ts

```diff
@@ -168,7 +168,7 @@ describe('src/cypress/dom/visibility', () => {
       }
 
       // ensure all tests run against a scrollable window
-      const scrollThisIntoView = add('<div style=`height: 1000px;` /><div>Should be in view</div>')
+      const scrollThisIntoView = add('<div style=`height: 1000px;`></div><div>Should be in view</div>')
```

Comment:
jQuery no longer gracefully handles DOM that is not valid - so this would no longer create the accurate DOM to test against. This was just written wrong at some point.